exchanges/tests/test_kraken.py

```diff
@@ -50,3 +50,47 @@ def test_latest_ticker(self):
 
         # Querying a non existent asset pair should fail.
         self.assertRaises(RuntimeError, lambda: kraken.latest_trade('FAKE'))
+
+
+    def test_trades_history(self):
+        """Test the trades_history method"""
+
+        kraken = Kraken()
+
+        asset = 'XXBTZEUR'
+        since = 1470092400  # 2016-08-02 01:00:00 - unix timestamp
+        trades_history = kraken.trades_history(asset, since)
+
+        # The first was filled shortly after 'since'
+        tolerance = 10  # 10ms in unix timestamp
+        self.assertTrue(trades_history[0].time - since < tolerance)
+        self.assertEqual(len(trades_history), 1000)
+
+        # The rest are ordered
+        for i, trade in enumerate(trades_history[1:], 1):
+            self.assertTrue(trade.time > trades_history[i-1].time)
+
+        # By looking at /Trades?pair=XXBTZEUR&since=1470092400000000000
+        # in a web browser we can create a specific test with only
+        # 3 orders between 'since' and 'until'
+        until = 1470092421.1177
+        trades_history = kraken.trades_history(asset, since, until)
```

Comment:
Here, could you check that you have the correct prices and volumes? This is not verified anywhere in the tests.